Vic2ToHoI4/Source/HOI4World/HoI4Country.h

```diff
@@ -69,8 +69,13 @@ class Country
 		 const Regions& regions,
 		 Mappers::GraphicsMapper& graphicsMapper,
 		 Names& names);
+	explicit Country(const std::string& region_,
+		 const Regions& regions,
+		 Mappers::GraphicsMapper& graphicsMapper,
+		 Names& names);
```

Comment:
Maybe put a comment that this is used to build unrecognized nations, otherwise looks like a generic constructor.

Reply:
Done